### DISCUSSION TITLE
A11y

### DISCUSSION THREAD

**Discussion by abarnes07** [themesberg/flowbite-vue#167] [Q&A] [answered]
Hello,

I recently attempted using flowbite and flowbite-vue in a Nuxt 3 project as it seemed to have a large amount of nicely designed components. Unfortunately, none of the components were a11y compliant which was a requirement for the site being worked on, so I ended up basically using flowbite in custom components. I'm just wondering if there are any plans to make the components a11y compliant in future releases or if that's on a roadmap somewhere?

**Comment by cogor** [comment, marked as answer]
Hello, of course we have plans to work on accessibility
